packages/liquidsoap/liquidsoap.2.0.0~rc1/opam

```diff
@@ -0,0 +1,190 @@
+  "ffmpeg-avcodec" {< "1.0.0~beta2"}
+  "ffmpeg-avdevice" {< "1.0.0~beta2"}
+  "ffmpeg-avfilter" {< "1.0.0~beta2"}
+  "ffmpeg-swresample" {< "1.0.0~beta2"}
+  "ffmpeg-swscale" {< "1.0.0~beta2"}
+  "flac" {< "0.3.0"}
+  "frei0r" {< "0.1.0"}
+  "gstreamer" {< "0.3.1"}
+  "inotify" {< "1.0"}
+  "ladspa" {< "0.2.0"}
+  "lame" {< "0.3.4"}
+  "lastfm" {< "0.3.0"}
+  "lo" {< "0.2.0"}
+  "mad" {< "0.5.0"}
+  "magic" {< "0.6"}
+  "ogg" {< "0.7.0"}
+  "opus" {< "0.2.0"}
+  "portaudio" {< "0.2.0"}
+  "posix-time2"
+  "pulseaudio" {< "0.1.4"}
+  "samplerate" {< "0.1.5"}
+  "sdl-liquidsoap" {< "2"}
+  "shine" {< "0.2.0"}
+  "soundtouch" {< "0.1.9"}
+  "speex" {< "0.4.0"}
+  "srt" {< "0.1.2"}
+  "ssl" {< "0.5.2"}
+  "taglib" {< "0.3.0"}
+  "tsdl-image" {< "0.2.0"}
+  "theora" {< "0.4.0"}
+  "vorbis" {< "0.8.0"}
+  "xmlplaylist" {< "0.1.3"}
+]
+build: [
+  ["./bootstrap"] {dev}
+  [
+    "./configure"
+    "--prefix"
+    prefix
+    "--sbindir=%{lib}%/liquidsoap/sbin"
+    "--libexecdir=%{lib}%/liquidsoap/libexec"
+    "--sysconfdir=%{lib}%/liquidsoap/etc"
+    "--sharedstatedir=%{lib}%/liquidsoap/com"
+    "--localstatedir=%{lib}%/liquidsoap/var"
+    "--libdir=%{lib}%/liquidsoap/lib"
+    "--includedir=%{lib}%/liquidsoap/include"
+    "--datarootdir=%{lib}%/liquidsoap/share"
+    "--with-bash-completion-dir=%{lib}%/liquidsoap/etc/bash_completion.d"
+    "--with-user=dummy"
+    "--with-group=dummy"
+  ]
+  [make "clean"] {dev}
+  [make]
+]
+install: [make "install"]
+post-messages: [
+  """\
+We're sorry that your liquidsoap install failed. Check out our installation
+instructions at: https://www.liquidsoap.info/doc-%{version}%/install.html#opam
+for more information."""
+    {failure}
+  "✨ Congratulations on installing liquidsoap! ✨" {success}
+  """\
+We noticed that you did not install any mp3 decoder. This is a feature most
+users want. You might need to install the mad or ffmpeg package."""
+    {success & !mad-enabled & !ffmpeg-decoder-enabled}
+  """\
+We noticed that you did not install any mp3 encoder. This is a feature most
+users want. You might need to install the lame or shine package."""
+    {success & !lame-enabled & !shine-enabled & !ffmpeg-encoder-enabled}
+  """\
+We noticed that you did not install the taglib package that provides support
+for reading metatadata in audio files. This is a feature most users want."""
+    {success & !taglib-enabled}
+  """\
+We noticed that you did not install the samplerate package. We strongly
+recommend this package for audio samplerate conversion."""
+    {success & !samperate-enabled}
+  """\
+We noticed that you did not install the cry package that provides icecast
+output. This is a feature most users want."""
+    {success & !cry-enabled}
+  """\
+We noticed that you did not install any ssl support package. Liquidsoap won't
+be able to use any HTTPS feature. You might want to install one of ssl or
+osx-secure-transport package."""
+    {success & !ssl-enabled & !secure-transport-enabled}
```

Comment:
mh, actually, what are those `*-enabled` variables? Did you mean `*:enabled` ?

Reply:
or rather `*:installed` most likely?

Reply:
We generate those variable as part of the `configure` script and use them to make the installation more user friendly (when users actually read those!)

Reply:
```
% cat liquidsoap.config
opam-version: "2.0"
variables {
  vorbis-enabled: true
  theora-enabled: true
  speex-enabled: true
  flac-enabled: true
  ogg_flac-enabled: true
  mad-enabled: true
  faad-enabled: true
  ffmpeg-avutil-enabled: true
  ffmpeg-av-enabled: true
  ffmpeg-avcodec-enabled: true
  ffmpeg-avdevice-enabled: true
  ffmpeg-avfilter-enabled: true
  ffmpeg-swresample-enabled: true
  ffmpeg-swscale-enabled: true
  ffmpeg-decoder-enabled: true
  ffmpeg-encoder-enabled: true
  ffmpeg-liq-filter-enabled: true
  xmlplaylist-enabled: true
  lastfm-enabled: true
  lame-enabled: true
  shine-enabled: false
  fdkaac-enabled: true
  opus-enabled: true
  taglib-enabled: true
  camomile-enabled: true
  cry-enabled: true
  ao-enabled: true
  ogg-enabled: true
  alsa-enabled: false
  portaudio-enabled: true
  pulseaudio-enabled: true
  bjack-enabled: true
  gstreamer-enabled: false
  samplerate-enabled: true
  soundtouch-enabled: true
  ladspa-enabled: false
  frei0r-enabled: true
  camlimages-enabled: false
  dssi-enabled: false
  graphics-enabled: true
  sdl-enabled: false
  sdl-image-enabled: false
  sdl-ttf-enabled: false
  posix-time2-enabled: false
  gd-enabled: false
  inotify-enabled: false
  lo-enabled: true
  magic-enabled: false
  yojson-enabled: true
  ssl-enabled: true
  secure_transport-enabled: false
  srt-enabled: true
  winscv-enabled: false
  prometheus-enabled: true
}
```